### ISSUE TITLE
Final WCAG issues

### ISSUE THREAD

**Issue by EdoPlantinga** [minvws/nl-covid19-notification-app-website#91]

**Comment by erikkroes** [comment]
Wat willekeurige observaties

- De slider is een bron van problemen
- De modals zijn niet echt modal

- Links hebben geen underline
- aXe vindt 5 issues
- De tekst "meer uitleg" is aangevuld
- alt-attributes en link-teksten kunnen scherper

**Comment by mellewynia** [comment]
Huidige stand van zaken van toegankelijkheidsverklaring (cc @bramd, @erikkroes ):
https://minvws.github.io/nl-covid19-notification-app-website/toegankelijkheidsverklaring

**Comment by gberkouwer** [comment]
Is bekend dat dit een standaard verklaring is die VWS moet invullen en ondertekenen, conform https://www.toegankelijkheidsverklaring.nl/? Die moet dan bij livegang een plek krijgen, bv onder /toegankelijkheid. Zie voorbeeld op https://www.rijksoverheid.nl/toegankelijkheid

**Comment by mellewynia** [comment]
@alexdekok , zie opmerking van @gberkouwer .

**Comment by alexdekok** [comment]
@mellewynia  Hier heeft @MartinW-com vanmiddag een opmerking over gemaakt richting mij. Voor de soft launch hanteren we de oplossing zoals nu is opgezet.

**Comment by bramd** [comment]
@MartinW-com @alexdekok Wat opmerkingen over de verklaring:

* Vul binnenkort een verklaring in via toegankelijkheidsverklaring.nl en maak van onze eigen toegankelijkheidspagina een simpele pagina die hiernaar linkt en toelichting geeft over toegankelijkheid en een contactmogelijkheid
* De datum waarop de verklaring voor het laatst is gewijzigd is verplicht, voeg deze toe aan onze tijdelijke verklaring
* De tijdelijke verklaring linkt naar een contactformulier van "ontregel de zorg", dat lijkt mij niet de juiste plaats voor toegankelijkheidsklachten over deze website? Als dit wel de juiste plaats is, is het voor de gebruiker alsnog zeer onlogisch om hier uit te komen
  * Een eigen contactformulier in deze statische website bouwen is te veel werk, dus het lijkt me beter om een e-mailadres te vermelden
  * Het aanmaken van een Github issue is in dit project natuurlijk ook een goede mogelijkheid voor toegankelijkheidsmeldingen voor de mensen die daar bekend/comfortabel mee zijn
* De tijdelijke verklaring stelt dat er maatregelen genomen zijn om toegankelijkheid te borgen. Dat is natuurlijk zo, maar loop deze lijst goed na en vraag je af of hij klopt, dit is niet iets om klakkeloos te copy-pasten
  * Bedenk dat het project publiek is, inclusief de ontwikkelgeschiedenis. Ik denk dat je op basis van die publieke geschiedenis kunt stellen dat er *niet* vanaf het begin toegankelijk gewerkt is
  * Als er toegankelijk ontworpen is, waarom zitten we dan nu nog met zoveel issues rondom de slider?
  * Een project dat niet perfect gestart is is geen ramp, maar wees daar eerlijk over in een dergelijke verklaring
* Er staat dat er voor september een onafhankelijk onderzoek zal plaatsvinden. Uit ervaring kan ik zeggen dat de markt zeer overspannen is door de deadline voor overheden in september. Als je voor die tijd een onderzoek wil, regel dat dan nu al, anders is dit niet meer haalbaar. Ik kan vanuit het team een audit volgens WCAG-EM doen en dat is afdoende om aan het beleid te voldoen. Als dit gewenst is, neem hier dan even contact over op om de planning af te stemmen. Ik kan ook andere partijen noemen voor een dergelijke audit.

**Comment by alexdekok** [comment]
@bramd dankjewel voor je opmerkingen.

- Je opmerking over het formulier en de verkeerde doorverwijzing. Pakken we direct op.
- Datum toevoegen verklaring doen we ook direct.

Overige punten komen we op terug.

**Comment by gberkouwer** [comment]
Ik voeg graag iets toe in lijn met de punten van @MartinW-com. Inmiddels is duidelijk dat de slider problemen oplevert bij livegang. Dat is exact waarom je tijdens de realisatie dit soort dingen meteen bekijkt :-). Het is dus een verstandige maatregel om de slider bij livegang gewoon weg te laten. Waarom heb je anders getest of hij wel of niet voldoet? Als de content in de slider van groot belang is, kan die ongetwijfeld een plek krijgen elders op de site. Daarmee is de website met een simpele actie toegankelijker geworden. Ik vermoed ook dat er geen gebruikersonderzoek is waaruit blijkt dat bezoekers de functionaliteit 'slider' onmisbaar vinden voor deze informatieve website? Weglaten helpt dus iedereen.

**Comment by HarrieKuipers** [comment]
Er zijn nog een aantal punten die aandacht nodig hebben. Er een slack channel aangemaakt voor discussie over deze issues. Met dank aan de community voor het signaleren en het aandragen van oplossingen.

**Comment by EdoPlantinga** [comment]
Van bovenstaande punten zijn er al een hoop opgepakt. De resterende issues zet @mellewynia op de backlog en hij zal ze oppakken. Hartelijk dank voor de suggesties tot zo ver, daar wordt het beter van.

**Comment by NL-William** [comment]
De HTML code voldoet nog niet helemaal aan de HTML-standaard, aldus https://validator.w3.org/nu/?doc=https%3A%2F%2Fcoronamelder.nl. Mogelijk zijn daar nog enkele issues mee te verhelpen (issues zien eruit als quick-wins).

Voorbeeldpunt:

 An img element must have an alt attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.
From line 83, column 11; to line 87, column 11
          <img↩            src="./img/phone.svg"↩            aria-hidden="true"↩            class="app-preview"↩          >↩     

Kwestie van in dit geval _alt="Telefoonicoon"_ toevoegen.

**Comment by StommePoes** [comment]
@NL-William er is geen nut om een gevulde `alt` te geven aan iets die niet voorkomt in de accessibility tree van de browser. "Telefoonicoon" zou nooit voorkomen in assistive technologies. Wel is het idd een validatie fout om geen attribuut te hebben.

Ik vind wel dat je op de standaard manier een decoratie laat zien als decoratie: s/aria-hidden="true"/alt=""
